### DISCUSSION TITLE
Developers Needed to Make Fun Educational Games

### DISCUSSION THREAD

**Discussion by sampolz** [community/community#29205] [Other Feature Feedback, Questions, & Ideas]
Hey everyone!

My name is Sam and I represent Legends of Learning - a platform for game based learning in schools, and a marketplace for game developers to earn revenue creating educational games. While Legends of Learning is an educational game company, we operate on the assumption that current educational games are not very good nor fun. Through an innovative marketplace of educational games, Legends of Learning teaches math and science to nearly 10 million students in over 30,000 schools (and growing fast). We are looking to work with passionate game studios and developers interested in revenue share to build FUN games that also happen to teach.

Games on our platform usually consist of 10-15 minutes of gameplay to teach a specific academic topic, and developers have complete creative freedom (really!) in how that topic is taught (e.g. you can create a tower defense game to teach “Heat as Energy Transfer” or a match-3 style game to teach “Chemical Reactions in Cellular Respiration”). The median new game will earn $8-13k in 2022 (annually), with this number rising each successive year ($15k+ in 2023, $20k+ in 2024, etc.). The longer your game stays on the platform, the more you will earn, and some devs have multiple games on the platform so their total earnings can be much higher.

Whether you are an independent developer or a 100-person studio, we want to work with you. Unlike other platforms that only make a lucky few studios wealthy in a "winner take all" marketplace, Legend's innovative model provides significant and growing payouts to the MEDIAN developer, with great payouts for top developers. Does building games that matter sound interesting to you? Visit **(removed by moderator)** to apply!

**Comment by queenofcorgis** [comment]
Hi @sampolz, we're happy you're here 😀
However, we currently do not allow job listings on the Comm
unity on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!
